Fusion freq README update (#100)
PediatricOpenTargets/OpenPBTA-analysis: analyses/fusion-frequencies/README.md

#### Frequency annotation

For each `cancer_group` and `cohort`(s) combination is considered a `cancer_group_cohort`. `cancer_group_cohort` with `n_samples` >= 5,  `Frequency_in_overall_dataset`, `Frequency_in_primary_tumors`, and `Frequency_in_relapse_tumors` were computed. 

> In this module, we generated two fusion frequency tables - one `putative-oncogene-fusion-freq.jsonl.gz` calculate the frequency of a specific fusion (with both partners) in primary tumors, relapse tumors and overall dataset. 

Reviewer: ```suggestion
In this module, we generated two fusion frequency tables - one `putative-oncogene-fusion-freq.jsonl.gz` calculate the frequency of a specific fusion in primary tumors, relapse tumors, and overall dataset. 
```

just updating this since one breakpoint or both can be intergenic (and a big fan of the oxford comma 😆 )

> `putative-oncogene-fused-gene-freq.jsonl.gz` on the other hand, calculate the frequency of a particular gene being fused in primary tumors, relapse tumors and overall dataset.

Reviewer: ```suggestion
In `putative-oncogene-fused-gene-freq.jsonl.gz` on the other hand, we calculate the frequency of a particular gene being fused in primary tumors, relapse tumors, and overall dataset.
```

> As long as a gene is a partner in a particular fusion (either 1A, 1B, 2A or 2B), it woudl be count as fused in that particular sample. 

Reviewer: ```suggestion
As long as a gene is a partner in a particular fusion (from 1A, 1B, 2A or 2B), it is counted as fused in that particular sample. 
```

  - Count the total number of samples in the `cancer_group_cohort` that are also in the `../independent-samples/results/independent-specimens.rnaseq.primary.eachcohort.tsv`, and call this number `Primary_tumors_in_dataset`.
  - `Frequency_in_primary_tumors = Total_primary_tumors_alterated / Primary_tumors_in_dataset`.

- `Frequency_in_relapse_tumors`:
  - For each unique fusion, count the number of samples (identified by `Kids_First_Biospecimen_ID`) that are in the `../independent-samples/results/independent-specimens.rnaseq.relapase.eachcohort.tsv`, and call this number `Total_relapse_tumors_alterated`.
  - Count the total number of samples in the `cancer_group_cohort` that are also in the `../independent-samples/results/independent-specimens.rnaseq.relapase.eachcohort.tsv`, and call this number `Relapse_tumors_in_dataset`.
  - `Frequency_in_relapse_tumors = Total_relapse_tumors_alterated / Relapse_tumors_in_dataset`.

Format the SNV mutation frequency table according to the latest spreadsheet that is attached in <https://github.com/PediatricOpenTargets/ticket-tracker/issues/64>.

Merge the fusion frequency tables of all `cancer_group_cohort`s.

> The combined table was then written out as `putative-oncogene-fusion-freq.jsonl.gz`.
> 
> Table two - `putative-oncogene-fused-gene-freq.jsonl.gz`
> The overall calculation logic is identical to table one. 
> The difference is that: instead of calculating the frequency for each unique fusion, we are calculating the frequency for each unique gene that are being fused (as either 1A, 1B, 2A or 2B in a particular fusion) in different tumor types.

Reviewer: ```suggestion
The combined table was is written out as `putative-oncogene-fusion-freq.jsonl.gz`.

Table two - `putative-oncogene-fused-gene-freq.jsonl.gz`
The overall calculation logic is identical to table one. 
The difference is instead of calculating the frequency for each unique fusion, we are calculating the frequency for each unique gene that is present within a fusion (from 1A, 1B, 2A or 2B in a particular fusion) in different tumor types.
```
